app/views/users/index.html.erb

```diff
@@ -1,29 +1,14 @@
 <h1>Here is the list of Users</h1>
-<article>
-  <h2>User 1</h2>
-  <p>Number of post: 4</p>
-</article>
-<article>
-  <h2>User 2</h2>
-  <p>Number of post: 5</p>
-</article>
-<article>
-  <h2>User 3</h2>
-  <p>Number of post: 3</p>
-</article>
-<article>
-  <h2>User 4</h2>
-  <p>Number of post: 6</p>
-</article>
-<article>
-  <h2>User 5</h2>
-  <p>Number of post: 2</p>
-</article>
-<article>
-  <h2>User 6</h2>
-  <p>Number of post: 1</p>
-</article>
-<article>
-  <h2>User 7</h2>
-  <p>Number of post: 0</p>
-</article>
\ No newline at end of file
+<% @users.each do |user| %>
+<article class='user'>
+  <figure>
+    <img class='user-image' src= '<%= user.photo %>' alt="<%= user.name %>" />
+  </figure>
+  <%= link_to user_path(user.id) do  %>
+    <section class='user-info'> 
+      <h2>User: <%= user.name %></h2>
+      <p>Number of post:<strong> <%= user.post_counter || 0 %> </strong> </p>
```

Comment:
The post conter for posts on the User Index page is not working correctly. Even though i have a post created for user, it is still displaying 0. Kindly fix this issue. 
![image](https://github.com/LarryIVC/larrys-blog/assets/33268927/357c1cd7-4464-4574-9211-5c1746938c94)

Reply:
Done